### PR TITLE
Corrected line for creating new namespace and added unit tests

### PR DESCRIPTION
**Pull request by KirbyKatcher**
[#1688](https://github.com/galasa-dev/projectmanagement/issues/1688)

### CODE REVIEW

**Comment by galasa-team** [comment]
Build successful

**Comment by galasa-team** [comment]
Build successful

**Comment by galasa-team** [comment]
Build successful

File: galasa-parent/dev.galasa.framework.api.common/src/main/java/dev/galasa/framework/api/common/resources/CPSFacade.java [hunk 1]
```diff
@@ -50,7 +50,7 @@ public CPSNamespace getNamespace(String name) throws ConfigurationPropertyStoreE
         getNamespaces();
         CPSNamespace namespace = bakedInNamespaceMap.get(name);
         if (namespace == null) {
-            new CPSNamespace(name, Visibility.NORMAL ,this.framework);       
+            namespace = new CPSNamespace(name, Visibility.NORMAL ,this.framework);
```
Comment (techcobweb) [inline galasa-parent/dev.galasa.framework.api.common/src/main/java/dev/galasa/framework/api/common/resources/CPSFacade.java:53]: lol. Surprised this wasn't picked up in a unit test originally.
Obvious now I see you change it,.

**Review by techcobweb — APPROVED** [review]
looks good

**Pull request merged**